src/core/Akka/IO/TcpSettings.cs

```diff
@@ -12,9 +12,9 @@
 namespace Akka.IO
 {
     /// <summary>
-    /// TBD
+    /// Settings for Akka.IO.Tcp's outbound and inbound connection acvtors.
     /// </summary>
-    public class TcpSettings
+    public sealed record TcpSettings
```

Comment:
BREAKING CHANGE: made this into a `sealed record` - should be binary compatible as long as people weren't inheriting from it.